docker/prod/docker_staging.env

```diff
@@ -55,3 +55,10 @@ EKS_NODEGROUP_ROLE_ARN=x
 SLACK_WEB_HOOK_URL=x
 ENVIRONMENT=staging
 SERVICE_DNS=x
+
+GF_SECURITY_ADMIN_USER=x
+GF_SECURITY_ADMIN_PASSWORD=x
+
+prometheus_multiproc_dir=/code/monitoring/prometheus/django
+
+PUSHGATEWAY_ENDPOINT=https://staging.eval.ai/pushgateway
```

Comment:
Add newline. Don't hardcode the endpoint here. We don't check in endpoints in `_staging` or `_prod` `.env` files

Reply:
@Ram81 how should I store the url for pushgateway? i would need to access it in the submission_worker.py

Reply:
Just mark it x for now. Env files are pulled from S3 where we store original values